ksql-engine/src/main/java/io/confluent/ksql/services/KafkaTopicClientImpl.java

```diff
@@ -176,6 +185,11 @@ public boolean isTopicExists(final String topic) {
     } catch (final ExecutionException e) {
       throw new KafkaResponseGetFailedException(
           "Failed to Describe Kafka Topic(s): " + topicNames, e.getCause());
+    } catch (final TopicAuthorizationException e) {
+      throw new TopicAuthorizationException(
+          AclErrorMessageUtil.generateErrorMessage(
+              AclOperation.DESCRIBE, StringUtils.join(topicNames, ','))
```

Comment:
nit: it's a little odd that the error message here will be
```
Failed to Describe Kafka topic: [topic1,topic2,topic3]
```
in the case of multiple topics. How about adding another method to `AclErrorMessageUtil` to generate error messages for multiple topics? Let's also add spaces after the commas.